src/components/Settings/Notifications/NotificationsTelegram.tsx

```diff
@@ -23,6 +23,9 @@ const messages = defineMessages({
   chatId: 'Chat ID',
   chatIdTip:
     'Start a chat with your bot, add <GetIdBotLink>@get_id_bot</GetIdBotLink>, and issue the <code>/my_id</code> command',
+  messageThreadId: 'Thread/Topic ID',
+  messageThreadIdTip:
+    "If your group chat has topics enabled, you can specify a thread/topic's ID here",
```

Comment:
Just a note, some of these are camel case, some of them are just all lower case. Will probably want a standard here. I have noticed this in other files too. I don't really care what the standard is, nor do I have the power to say what it should be, just making an observation.

Reply:
Good catch, thanks! I was just following the convention of the neighboring variables. 
Please lmk if that needs to be adjusted.